src/wp-includes/functions.php

```diff
@@ -5195,7 +5195,7 @@ function wp_widgets_add_menu() {
 		return;
 	}
 
-	$submenu['themes.php'][7] = array( __( 'Widgets' ), 'edit_theme_options', 'widgets.php' );
+	$submenu['themes.php'][8] = array( __( 'Widgets' ), 'edit_theme_options', 'widgets.php' );
```

Comment:
Question: What will the impact be to extenders who expect the Widgets menu to be in the 7th position as it's always been? Changing this might cause another problem for others.